app/controlplane/internal/conf/controlplane/config/v1/conf.proto

```diff
@@ -58,6 +58,9 @@ message Bootstrap {
     // https://github.com/chainloop-dev/chainloop/blob/126f47b6c0803eac844b8e3e1a21d582f00e4dc6/app/artifact-cas/internal/service/download.go#L34
     string download_url = 3;
   }
+
+  // Configuration for onboarding users in organizations with specific roles
+  OnboardingSpec onboarding = 10;
```

Comment:
any reason you want to have a two level object here? 

```
onboarding:
  `auto_onboard_organizations`: ...
```

wouldn't something like

`repeated OnboardingSpec auto_onboarding = 10` be enough?

That will render a simpler yaml I think

```yaml
auto_onboarding:
  - name: foo
    role: bar
```

Reply:
No strong reason, just made it two object in case we need to add more functionalities in the future, we have the `OnboardinSpec` already there.

Reply:
In any case, I can flatten it, not a big deal.